meinberlin/apps/platformemails/migrations/0005_alter_platformemail_body.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.9 on 2024-01-30 09:17
+
+import adhocracy4.images.validators
+from django.db import migrations
+import django_ckeditor_5.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("meinberlin_platformemails", "0004_alter_platformemail_body"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="platformemail",
+            name="body",
+            field=django_ckeditor_5.fields.CKEditor5Field(
+                blank=True,
+                help_text="When adding images, please ensure to set the width no larger than 650px.",
+                validators=[adhocracy4.images.validators.ImageAltTextValidator()],
+                verbose_name="Email body",
+            ),
+        ),
+    ]
```

Comment:
help_text can go to PlatformEmailForm, 

squashing those later.

Reply:
is done in #5412 already